src/App.tsx

```diff
@@ -1,8 +1,117 @@
-import React from 'react';
+import React, {useState, useEffect, useContext} from 'react';
 import logo from './logo.svg';
 import nasaLogo from './NASA.svg';
 import './App.css';
 
+function Count(){
+  let countText = localStorage.getItem('count');
+  let countInt: number = 0;
+  if(countText == null){
+    countInt = 0;
+  }
+  else {
+    countInt = parseInt(countText);
+  }
+  const [count,setCount] = useState(countInt);
+
+  useEffect(() => {
+    localStorage.setItem('count',count.toString());
```

Comment:
nitpick but we should add spaces after commas. Can you please do that everywhere in the file?